src/breeze/actions/mixins/action_helpers.cr

```diff
@@ -23,6 +23,24 @@ module Breeze::ActionHelpers
         )
       end
     end
+
+    Lucky::Events::PipeEvent.subscribe do |event|
+      next unless Breeze.settings.enabled
+      # TODO: move this to a config setting
+      next if event.name.includes?("breeze_")
```

Comment:
I dont think this will generally conflict it any other pipes but what about this.
```suggestion
      next if event.name.starts_with?("breeze_")
```

Reply:
The names are actually `store_breeze_request` and `store_breeze_response`. https://github.com/luckyframework/breeze/pull/36/files#diff-93ceec79cd40984847897f3a614ee23b806a0a1ac240bd153a00bff4b4a892efR7-R8

I guess I could do `starts_with?("store_breeze_")` ... 🤔

Reply:
I think this is a very unlikely edge case.  but I think `starts_with?("store_breeze_")` might be a good idea if.